cmd/conduit/main.go

```diff
@@ -155,7 +162,37 @@ func runConduitInit(path string) error {
 	}
 	defer f.Close()
 
-	_, _ = f.WriteString(sampleConfig)
+	var processorList []string
+
+	if processorsFlag != "" {
+		processorList = strings.Split(processorsFlag, ",")
+	}
+
+	var importer string
+	const twoSpaces = "  "
+	if importerFlag == "" {
+		importer = defaultImporter
+	} else {
+		importer = twoSpaces + "name: " + importerFlag + "\n" + twoSpaces + "config:"
+	}
+
+	var exporter string
+	if exporterFlag == "" {
+		exporter = defaultExporter
+	} else {
+		exporter = twoSpaces + "name: " + exporterFlag + "\n" + twoSpaces + "config:"
+	}
+
+	var processors string
+	for _, processor := range processorList {
+		processors = processors + twoSpaces + "- "
+		processors = processors + "name: " + processor + "\n" + twoSpaces + twoSpaces + "config:\n"
+	}
```

Comment:
could you add:
* validation - make sure the requested plugin exists.
* use the metadata template to fill in a sample config.

There are some helpers to fetch the metadata:
https://github.com/algorand/indexer/blob/develop/conduit/pipeline/metadata.go

The `./conduit list ...` command shows this data being used:
https://github.com/algorand/indexer/blob/develop/cmd/conduit/internal/list/list.go